packages/forma-36-react-components/src/components/Modal/ModalConfirm/ModalConfirm.stories.tsx

```diff
@@ -102,9 +116,9 @@ export function ComplexStory(props: ModalConfirmProps) {
         }}
         {...props}
       >
-        <p>
+        <div>
```

Comment:
It's totally minor, but we could have a margin-bottom here I think.

Reply:
added Paragraph